types/make-fetch-happen/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
@MichaelDeBoey: Perhaps you could help enlighten me on what the best practice is here! I'd originally gone with this approach for putting `dom` in the `tsconfig.json` rather than the triple slash lib approach.  However, https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50588 was raised soon afterward and my validations showed that PR to be indeed necessary.  Most importantly, the need for consumers to include `dom` in their own `tsconfig.json` merely because they're using these types seems undesirable and that PR resolved that.  (Also, the `README` seemed to convince me that was the correct approach, but perhaps that's changed?)

Curious in your guidance, @MichaelDeBoey!

Reply:
I'm used to having it in `lib`, as that's what I see in other libraries.
Reverted it.